types/email-templates/email-templates-tests.ts

```diff
@@ -1,29 +1,14 @@
-import EmailTemplates = require('email-templates');
+import * as Email from 'email-templates';
```

Comment:
Please keep this as `import EmailTemplates = require('email-templates')` given that the other file uses the `export =` form.

Reply:
done! thanks